specutils/analysis/flux.py

```diff
@@ -116,3 +118,57 @@ def _compute_equivalent_width(spectrum, continuum=1, regions=None):
     ew =  dx - (line_flux / continuum)
 
     return ew.to(calc_spectrum.spectral_axis.unit)
+
+def is_continuum_near_zero(spectrum, eps=0.01):
+    """
+    Determine if the baseline continuum of the spectrum is near zero.
+
+    The value is scaled to match the sigma/standard deviation parameter of a
+    standard Gaussian profile. This will be calculated over the regions, if
+    they are specified.
+
+    Parameters
+    ----------
+    spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
+        The spectrum object over which the width will be calculated.
+
+    eps: float
+        The tolerance on the quantification to confirm the continuum is
+        near zero.
+
+    Returns
+    -------
+    is_near_zero: bool
+        A True if the continuum of the spectrum is with eps, False otherwise.
+
+    Notes
+    -----
+    The spectrum should be continuum subtracted before being passed to this
```

Comment:
I think this note is not necessary, since the point is to check if this is true or not.

Reply:
k